### DISCUSSION TITLE
Repository sourced applications based on compose

### DISCUSSION THREAD

**Discussion by ljnsn** [coollabsio/coolify#1472] [Feature Requests]
On Coolify v4, I don't see any way to deploy applications sourced from a repository with a compose file. Is the only way to deploy applications with docker compose to paste the compose file into the "Based on Docker Compose" application option?

![image](https://github.com/coollabsio/coolify/assets/82611987/2b7df841-31ef-4357-a17c-8a0c0eefec81)

**Comment by andrasbacsai** [comment]
It is not yet supported, but coming soon. 🙂

**Reply by peaklabs-dev** [reply]
@ljnsn This has been added. Just select Docker Compose from the drop down menu.
